### PR TITLE
fix: correct the comment for resizeToFit

### PR DESCRIPTION
**Pull request by wy-luke**
According to the actual effect of `resizeToFit()`, I think it would be better to change the comment to `to fit it's content` rather `to fit container`.

The `TextBlock` will always be resized to fit it's content whether the text is too long or too large. Any excess beyond the container, like a plane attached the texture with one `TextBlock`,  will not be displayed.

In the meanwhile, when I set the `fontSize`, `width` and `resizeToFit` simultaneously, it's `width` to fit `fontSize`, rather `fontSize` to fit `width`.

```
let text = new BABYLON.GUI.TextBlock();
text.text = "Hello world";
text.fontSize = 50;
text.width = 0.5
text.resizeToFit = true
```

**It's all about the content.** There's nothing about the `container` or `to fit container`.

As far as I'm concerned, the original version is kind of ambiguous to understand. And I think it might be better to make this modification for better clarity.

Thanks.

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/14363/merge

Test environment: 
https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14363/merge/index.html

To test a playground add it to the URL, for example:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14363/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/14363/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/14363/merge
https://gui.babylonjs.com/?snapshot=refs/pull/14363/merge
https://nme.babylonjs.com/?snapshot=refs/pull/14363/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/14363/merge#BCU1XR#0

File: packages/dev/gui/src/2D/controls/textBlock.ts [hunk 1]
```diff
@@ -76,15 +76,15 @@ export class TextBlock extends Control {
     }
 
     /**
-     * Gets or sets an boolean indicating that the TextBlock will be resized to fit container
+     * Gets or sets an boolean indicating that the TextBlock will be resized to fit it's content
```
Comment (sebavan) [inline packages/dev/gui/src/2D/controls/textBlock.ts:79]:
```suggestion
     * Gets or sets an boolean indicating that the TextBlock will be resized to fit its content

```

File: packages/dev/gui/src/2D/controls/textBlock.ts [hunk 2]
```diff
@@ -76,15 +76,15 @@ export class TextBlock extends Control {
...
      */
     @serialize()
     public get resizeToFit(): boolean {
         return this._resizeToFit;
     }
 
     /**
-     * Gets or sets an boolean indicating that the TextBlock will be resized to fit container
+     * Gets or sets an boolean indicating that the TextBlock will be resized to fit it's content
```
Comment (sebavan) [inline packages/dev/gui/src/2D/controls/textBlock.ts:87]:
```suggestion
     * Gets or sets an boolean indicating that the TextBlock will be resized to fit its content

```

**Review by sebavan — APPROVED** [review]

**Pull request merged**